test/Unit/Runners/PHPUnit/ResultPrinterTest.php

```diff
@@ -113,6 +113,45 @@ public function testStartPrintsOptionInfoAndConfigurationDetailsIfConfigFilePres
         static::assertStringStartsWith($expected, $contents);
     }
 
+    public function testStartPrintsOptionInfoWithRandom(): void
+    {
+        $pathToConfig = TMP_DIR . DS . 'phpunit-myconfig.xml';
+
+        file_put_contents($pathToConfig, '<root />');
+        $this->printer = new ResultPrinter($this->interpreter, $this->output, $this->createOptionsFromArgv([
+            '--configuration' => $pathToConfig,
+            '--order-by' => Options::RANDOM_ORDER,
+            '--random-order-seed' => 1234,
+        ]));
+        $contents      = $this->getStartOutput();
+        $expected      = sprintf(
+            "Running phpunit in %s processes with %s\n\nConfiguration read from %s\n\n",
+            PROCESSES_FOR_TESTS,
+            $this->options->phpunit(),
+            $pathToConfig
+        );
+        static::assertStringStartsWith($expected, $contents);
```

Comment:
What is this test covering, exactly?
The assertion looks generic, without any ordering expectations

Reply:
Fixed